changelogs/fragments/67186_remove_deprecated_keys_nxos.yaml

```
@@ -0,0 +1,3 @@
+---
+removed_features:
+  - Removed previously deprecated keys `oif_prefix` and `oif_source` from nxos_igmp_interface (https://github.com/ansible/ansible/pull/67186).
```

```suggestion
  - Removed previously deprecated keys ``oif_prefix`` and ``oif_source`` from nxos_igmp_interface (https://github.com/ansible/ansible/pull/67186).
```
This is used in Restructured Text, not in Markdown :)

---

Oh, right. I'll fix it. Thanks for pointing it out.